music21/harmony.py

```diff
@@ -2355,6 +2380,22 @@ def writeAsChord(self):
     def writeAsChord(self, val):
         pass
 
+    def transpose(self: T, _value, *, inPlace=False) -> Optional[T]:
+        '''
+        Overrides :meth:`~music21.chord.Chord.transpose` to do nothing.
+
+        >>> nc = harmony.NoChord()
+        >>> nc.figure
+        'N.C.'
+        >>> nc.transpose(8, inPlace=True)
+        >>> nc.figure
+        'N.C.'
+        '''
+        if not inPlace:
+            return copy.deepcopy(self)
+        else:
+            return self
```

Comment:
in general, are we returning None when inPlace?

Reply:
whoops -- meant that.  Thank you!